apps/devhub/templates/devhub/email/new-key-email.ltxt

```diff
@@ -0,0 +1,7 @@
+The following API Key was added to your account:
+{{ key }}
+
+If you believe this key was created in error, you can remove the key and
+disable access at the following location:
+
+<a href="{{ url('devhub.api_key') }}">{{ url('devhub.api_key') }}</a>
```

Comment:
This needs to use `absolutify` so that it appears as an absolute URL. [Here's an example](apps/bandwagon/templates/bandwagon/includes/addedit.html).